great_tables/gt.py

```diff
@@ -363,6 +385,21 @@ def _render_as_html(self) -> str:
 </div>
         """
 
+        if make_page:
+
+            # Create an HTML page and place the table within it
+            finalized_table = f"""<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8"/>
+<style>body {{ background-color:white; }}
```

Comment:
This styles every body element on the page to white (including the overall body element for the page). We need a more specifically scoped css style. If we want to set only the body element defined here to white, then we need to either...

* specify the style inline
* give the body element a unique id and set the style on that

Reply:
Sorry, didn't go all the way with this. I'm going to remove it entirely.